### PR TITLE
Add flag for log-level

### PR DESCRIPTION
**Pull request by johscheuer**
This PR add the ability to specify the log-level when running kuberhealthy (e.g. only errors):

```bash
kuberhealthy - Kuberhealthy is an in-cluster synthetic health checker for Kubernetes.

  Flags: 
       --version  Displays the program version string.
    -h --help  Displays help with available flag, subcommand, and positional value parameters.
       --kubecfg  (optional) absolute path to the kubeconfig file
    -l --listenAddress  The port for kuberhealthy to listen on for web requests
       --componentStatusChecks  Set to false to disable daemonset deployment checking.
       --daemonsetChecks  Set to false to disable cluster daemonset deployment and termination checking.
       --podRestartChecks  Set to false to disable pod restart checking.
       --podStatusChecks  Set to false to disable pod lifecycle phase checking.
       --forceMaster  Set to true to enable local testing, forced master mode.
    -d --debug  Set to true to enable debug.
       --dsPauseContainerImageOverride  Set an alternate image location for the pause container the daemon set checker uses for its daemon set configuration.
       --podCheckNamespaces  The comma separated list of namespaces on which to check for pod status and restarts, if enabled.
       --log-level  Log level to be used one of [panic,fatal,error,warning,info,debug,trace].
```

Many changes are directly from `gofmt -s -w .`

### CODE REVIEW

File: cmd/kuberhealthy/main.go [hunk 1]
```diff
@@ -60,6 +62,15 @@ const CRDResource = "khstates"
 
 var masterCalculationInterval = time.Second * 10
 
+func getAllLogLevel() string {
```
Comment (integrii) [inline cmd/kuberhealthy/main.go]: fancy!

**Comment by integrii** [comment]
Thanks for the contribution @johscheuer !  We normally push things into the `unstable` branch first, so I changed your PR to be against unstable and resolved the conflicts that created.  This is well done and helpful.  Thanks again!

**Comment by termux-io** [comment]
https://docs.microsoft.com/en-us/azure/devops/boards/github/connect-to-github?view=azure-devops

**Comment by termux-io** [comment]
https://patch-diff.githubusercontent.com/raw/Comcast/kuberhealthy/pull/174.patch

**Comment by termux-io** [comment]
`0afaca578345b756eaf2fa0aaea9dfa0ef73de99`

**Pull request merged**